pkg/controller/login/register.go

```diff
@@ -16,17 +16,38 @@
 package login
 
 import (
+	"fmt"
 	"net/http"
 
+	"github.com/google/exposure-notifications-server/pkg/logging"
 	"github.com/google/exposure-notifications-verification-server/pkg/controller"
 )
 
 func (c *Controller) HandleRegisterPhone() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
+		logger := logging.FromContext(ctx).Named("login.HandleRegisterPhone")
+
+		session := controller.SessionFromContext(ctx)
+		if session == nil {
+			err := fmt.Errorf("session does not exist in context")
+			logger.Errorw("failed to get session", "error", err)
+			controller.InternalError(w, r, c.h, err)
+			return
+		}
+
+		realm := controller.RealmFromContext(ctx)
```

Comment:
instead of requiring a realm  - why not bounce the user back here after they select a realm that requires MFA and include that in the flash messages?

Then we can send all users here as a suggestion on first login

Reply:
because then they can just navigate to /home, this enforces it